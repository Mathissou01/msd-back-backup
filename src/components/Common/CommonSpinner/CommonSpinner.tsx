import React from "react";
import "./common-spinner.scss";

export default function CommonSpinner() {
  return (
    <div className="c-CommonSpinner" data-testid="common-spinner">
      <div />
      <div />
      <div />
      <div />
    </div>
  );
}
