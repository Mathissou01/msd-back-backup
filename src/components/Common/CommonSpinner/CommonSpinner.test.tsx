import { render } from "@testing-library/react";
import CommonSpinner from "./CommonSpinner";

describe("CommonSpinner", () => {
  it("renders", () => {
    const { container } = render(<CommonSpinner />);
    expect(container).toMatchSnapshot();
  });
});
