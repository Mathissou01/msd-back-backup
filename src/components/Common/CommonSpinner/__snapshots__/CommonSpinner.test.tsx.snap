// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonSpinner renders 1`] = `
<div>
  <div
    class="c-CommonSpinner"
    data-testid="common-spinner"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
